lib/katello/plugin.rb

```diff
@@ -19,8 +19,9 @@
     menu :top_menu,
          :redhat_provider,
          :caption => N_('Red Hat Repositories'),
-         :url_hash => {:controller => 'katello/providers',
```

Comment:
Thoughts on providing a redirect to the new page from katello/providers/redhat_provider? It's become a 404 now

Reply:
That's a good suggestion, added.